tests/sdk/nodejs/nodejs_test.go

```diff
@@ -271,6 +271,21 @@ func TestCRDs(t *testing.T) {
 	integration.ProgramTest(t, &test)
 }
 
+func TestCRDUpgrade(t *testing.T) {
+	test := baseOptions.With(integration.ProgramTestOptions{
+		Dir:                  filepath.Join("crd-upgrade", "step1"),
+		Quick:                false,
```

Comment:
```suggestion
		Quick:                false,
```
Remove this since its the default anyway?